packages/fxa-auth-server/scripts/recorded-future/check-and-reset.ts

```diff
@@ -102,122 +123,169 @@ Limit: ${resultsPerPageLimit}
     return 0;
   }
 
+  const accountsToReset = await (async () => {
+    if (!resetWithEmailArgs) {
+      return await findLeakedAccounts(firstDownloadedDateIsoString);
+    }
+
+    const loginsFromEmailArgs = program.email.map((x) => ({
+      login: x,
+      domain: searchDomain,
```

Comment:
Doesn't look like `domain` is used in the `getAccountsByLogin` function.

Reply:
👍 I'll update it in the next PR.